cypress/support/pages/home.pageObject.js

```diff
@@ -8,8 +8,15 @@ class HomePageObject extends PageObject {
   }
 
   assertHeaderContainUsername(username) {
-    this.usernameLink
-      .should('contain', username);
+    this.usernameLink.should('contain', username);
```

Comment:
You are using 'this' keyword in the function 'assertHeaderContainUsername'. Consider replacing 'this' keyword with arrow functions or Function#bind for better readability and maintainability.

Reply:
done

Reply:
Done